tests/test_wave_03.py

```diff
@@ -6,35 +6,24 @@
 import pytest
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
+#.mark.skip(reason="No way to test this feature yet")
 def test_mark_complete_on_incomplete_task(client, one_task):
     # Arrange
-    """
-    The future Wave 4 adds special functionality to this route,
-    so for this test, we need to set-up "mocking."
 
-    Mocking will help our tests work in isolation, which is a
-    good thing!
-
-    We need to mock any POST requests that may occur during this
-    test (due to Wave 4).
-
-    There is no action needed here, the tests should work as-is.
-    """
     with patch("requests.post") as mock_get:
         mock_get.return_value.status_code = 200
 
         # Act
-        response = client.patch("/tasks/1/mark_complete")
-
+        response = client.patch("/tasks/1/mark_complete", headers={"Content-Type": "application/json"})
+        print(response.status_code, response.text)
```

Comment:
@vfarinango, be mindful to only alter the tests that marked to be implemented.
```suggestion
        response = client.patch("/tasks/1/mark_complete")
```